downstream/modules/platform/proc-scm-git-subversion.adoc

```diff
@@ -0,0 +1,43 @@
+[id="proc-scm-git-subversion"]
+
+= SCM Types - Git and Subversion
+
+Use the following procedure to configure playbooks to use source control.
+
+.Procedure
+. In the Project *Details* tab, select the appropriate option (Git or Subversion) from the *SCM Type* menu.
++
+image:projects-create-scm-project.png[Select scm]
+
+. Enter the appropriate details into the following fields:
+
+* *SCM URL* - See an example in the tooltip .
+* Optional: *SCM Branch/Tag/Commit*: Enter the SCM branch, tags, commit hashes, arbitrary refs, or revision number (if applicable) from the source control (Git or Subversion) to checkout. 
+Some commit hashes and referencess may not be available unless you also provide a custom refspec in the next field. 
+If left blank, the default is `HEAD` which is the last checked out Branch, Tag, or Commit for this project.
+* *SCM Refspec* - This field is an option specific to git source control and only advanced users familiar and comfortable with git should specify which references to download from the remote repository. 
+For more information, see [job branch overriding].
```

Comment:
I've created the xref for this xref:controller-job-branch-overriding[Job branch overriding]